src/chef-mover/rebar.lock

```diff
@@ -7,10 +7,10 @@
   {git,"git://github.com/opscode/automeck.git",
        {ref,"363657b4dff5ef5561e77a7d44348abf11405d09"}},
   2},
- {<<"bcrypt">>,
-  {git,"git://github.com/smarkets/erlang-bcrypt.git",
-       {ref,"a63df34d4957dbb70a703c67c75ed9fee2c78971"}},
-  0},
+% {<<"bcrypt">>,
```

Comment:
Regenerating the lock file should pull that change pointing to https://github.com/abhay/erlang-bcrypt

Reply:
I'm assuming I can also just get the SHA off of the abhay branch?

By the way, found a reference to opscode/erlang-bcrypt in src/chef-mover/_checkouts/oc_erchef/rebar.config .

Reply:
Yep - that should also point to https://github.com/abhay/erlang-bcrypt the change to get bcrypt to work with rebar3 was what was done in our fork and that has been merged upstream.

Reply:
Will make this change now.